build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -118,16 +144,27 @@ public static BuildTargetResult GenerateCompressedFile(BuildTargetContext c)
         [BuildPlatforms(BuildPlatform.Windows)]
         public static BuildTargetResult GenerateZip(BuildTargetContext c)
         {
-            CreateZipFromDirectory(c.BuildContext.Get<string>("CombinedFrameworkHostRoot"), c.BuildContext.Get<string>("CombinedFrameworkHostCompressedFile"));
+            CreateZipFromDirectory(
+                c.BuildContext.Get<string>("CombinedMuxerHostFxrFrameworkPublishRoot"), 
+                c.BuildContext.Get<string>("CombinedMuxerHostFxrFrameworkCompressedFile"));
 
+            CreateZipFromDirectory(
+                c.BuildContext.Get<string>("HostFxrPublishRoot"), 
+                c.BuildContext.Get<string>("HostFxrCompressedFile"));
             return c.Success();
         }
 
         [Target(nameof(PackageTargets.InitPackage))]
         [BuildPlatforms(BuildPlatform.Unix)]
         public static BuildTargetResult GenerateTarBall(BuildTargetContext c)
         {
-            CreateTarBallFromDirectory(c.BuildContext.Get<string>("CombinedFrameworkHostRoot"), c.BuildContext.Get<string>("CombinedFrameworkHostCompressedFile"));
+            CreateTarBallFromDirectory(
+                c.BuildContext.Get<string>("CombinedMuxerHostFxrFrameworkPublishRoot"), 
+                c.BuildContext.Get<string>("CombinedMuxerHostFxrFrameworkCompressedFile"));
+
+            CreateTarBallFromDirectory(
```

Comment:
There has been no need so far for a muxer zip. The muxer will be installed with MSI to setup `PATH`. `hostfxr.dll` will be delivered through rapid update hence the zip up.